src/Documents/DocumentStore.ts

```diff
@@ -0,0 +1,60 @@
+import {IDocument} from './IDocument';
+import {IDocumentStore} from './IDocumentStore';
+import {IDocumentSession} from "./Session/IDocumentSession";
+import {DocumentSession} from "./Session/DocumentSession";
+import {RequestExecutor} from '../Http/RequestExecutor';
+import {DocumentConventions} from './Conventions/DocumentConventions';
+
```

Comment:
Let's use the following style conventions for Typescript code in this project:

Use PascalCase for class and interface names.
Use PascalCase for enum values.
Use camelCase for function names.
Use camelCase for property names and local variables.
Use camelCase for argument names.
Use "_" as a prefix for `private` properties (not for `protected` ones).
Use camelCase for file / module names;